src/core_ocean/mode_init/mpas_ocn_init_tidal_boundary.F

```diff
@@ -244,6 +252,15 @@ subroutine ocn_init_setup_tidal_boundary(domain, iErr)!{{{
           (config_tidal_boundary_right_bottom_depth - config_tidal_boundary_left_bottom_depth)
       end do
 
+      if (config_use_variable_drag) then
```

Comment:
@caozd999, here is the selection of different types of bottom drag you can use as an example to build your own transect and variable drag.

Reply:
https://github.com/MPAS-Dev/MPAS-Model/pull/389/files#diff-6270014bc9ae9b8554f8af50127caf5fL242 above shows bottom depth that needs modified.

I recommend building your own new `src/core_ocean/mode_init/mpas_ocn_init_tidal_transect.F` case.